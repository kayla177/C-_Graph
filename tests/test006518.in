ENTITY B1RQY2F6 name_B1RQ journal
ENTITY F1222OYH name_F122 conference
ENTITY HSDTVQXW name_HSDT paper
ENTITY 692SX4JI name_692S institution
ENTITY MM5NQE2R name_MM5N paper
ENTITY XWLTY346 name_XWLT journal
ENTITY X3E5MVWI name_X3E5 journal
ENTITY L8UDQXP0 name_L8UD institution
ENTITY 39OGBNRL name_39OG conference
ENTITY MV40PNUL name_MV40 institution
RELATIONSHIP MV40PNUL cited_by HSDTVQXW 0.8
RELATIONSHIP MV40PNUL collaborated_with F1222OYH 5.7
RELATIONSHIP B1RQY2F6 collaborated_with X3E5MVWI 8.0
RELATIONSHIP L8UDQXP0 presented_at 692SX4JI 9.0
RELATIONSHIP MM5NQE2R presented_at X3E5MVWI 1.7
RELATIONSHIP XWLTY346 cited_by B1RQY2F6 3.6
RELATIONSHIP MV40PNUL authored_by XWLTY346 4.5
RELATIONSHIP MV40PNUL presented_at B1RQY2F6 2.6
RELATIONSHIP XWLTY346 collaborated_with 39OGBNRL 2.4
RELATIONSHIP XWLTY346 published_in HSDTVQXW 8.8
RELATIONSHIP B1RQY2F6 published_in 39OGBNRL 6.4
RELATIONSHIP L8UDQXP0 presented_at F1222OYH 6.5
RELATIONSHIP XWLTY346 presented_at 692SX4JI 7.1
RELATIONSHIP F1222OYH authored_by MM5NQE2R 9.9
RELATIONSHIP HSDTVQXW cited_by B1RQY2F6 1.2
PATH MV40PNUL HSDTVQXW
DELETE MV40PNUL
DELETE HSDTVQXW
PRINT F1222OYH
PATH F1222OYH MM5NQE2R
PRINT F1222OYH
HIGHEST
DELETE 692SX4JI
PATH F1222OYH X3E5MVWI
PRINT 39OGBNRL
EXIT
