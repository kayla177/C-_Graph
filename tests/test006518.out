success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
MV40PNUL XWLTY346 HSDTVQXW 13.3
success
success
L8UDQXP0 MM5NQE2R
F1222OYH MM5NQE2R 9.9
L8UDQXP0 MM5NQE2R
MM5NQE2R XWLTY346 32.5
success
F1222OYH MM5NQE2R X3E5MVWI 11.6
XWLTY346 B1RQY2F6
